### ISSUE TITLE
Observable.once

### ISSUE THREAD

**Issue by fpillet** [RxSwiftCommunity/RxSwift-Ext#4]
### Name of the operator

`Observable.once()` or `Observable.justOnce()`

### Motivation for inclusion

The `Observable.just()` function creates a sequence that delivers a single value to its observers, then completes. In some cases you want an observable to deliver a value *once* and exactly once.

For example it may be a one-time location value for setting up the initial position of a map: you want the initial position to be forced the first time the map is displayed, but not subsequent times after the user has moved the map.

### Example of use

```swift
class MyViewController : UIViewController {
  @IBOutlet weak var mapView : MKMapView!

  let initialMapCenter : Observable<CLLocation> = Observable.justOnce(someLocation)

  func viewDidAppear(animated : Bool) {
    super.viewDidAppear(animated)
    _ = initialMapCenter.subscribeNext { [weak self] location in
      self?.mapView.setCenterCoordinate(location.coordinate, animated: true)
    }
  }
}
```

**Comment by icanzilb** [comment]
how would that observable play with `replay...`? Will always deliver that one value to all subscribers?

**Comment by fpillet** [comment]
It wouldn't replay. The purpose is to guarantee delivery once and only once, regardless of the number of subscribers and other operators applied to the sequence. Replay would just immediately terminate.

**Comment by fpillet** [comment]
Implemented by commit 3a8cc55
